### DISCUSSION TITLE
Dockerizing would lower the entry bar to new comers

### DISCUSSION THREAD

**Discussion by CrisFavero** [azimuttapp/azimutt#124] [Ideas]
Are there any plans/interest in docerkizing this tool? 
It looks really interesting, but I'll admit looking at needing to npm, elm, and elm-spa, especially with no notation of what versions it is known to work with of those, has given me hesitation from approaching this.  I will still likely try this out but wanted to know what plans there may be for making the local testing of this simpler.

**Comment by loicknuchel** [comment]
Hi @CrisFavero 
Indeed there is a lot of setup and we are not that good at documentation (and as we are adding an Elixir backend, it will not go easier ^^).
We are improving the setup commands but it will still be needed to install dependencies manually.
We are considering Docker but to be honest, I'm not that good with it so I don't really know how hard it would be, and, for now, I focus mainly on adding requested features by users.
If there is more interest in this, we could probably make this more high prio ;)

**Comment by sbouaked** [comment]
Small update, we are working on this subject with this PR https://github.com/azimuttapp/azimutt/pull/170
And you can check our first image here https://github.com/azimuttapp/azimutt/pkgs/container/azimutt
 
 Don't hesitate to test and share your feedback, I'm not a docker expert ! 😬
